.gitignore

```diff
@@ -10,4 +10,6 @@ packages/
 
 # JetBrains Rider
 *.iml
-.idea
\ No newline at end of file
+.idea
+/Sources/HipMobileUI.Android/Resources/drawable-hdpi/ic_my_location(500).png
```

Comment:
These files should be deleted instead of ignoring them

Reply:
Okay I shall delete the other files.
Only way to increase the resolution would be by adapting a bigger Icon. It will look huge if its okay then I could do it.

Reply:
Have made the necessary changes.

Reply:
Can you remove `/Sources/HipMobileUI.Android/Resources/drawable-hdpi/ic_my_location(500).png` from the .gitignore again?